test/util.go

```diff
@@ -23,6 +23,10 @@ import (
 	"github.com/knative/pkg/test/logging"
 )
 
+const (
+	HelloVolumePath = "/hello/world"
```

Comment:
Do we need to specify this as a const and use it in the image? Could we not pass a path to the application as a query parameter and the app then tries to read that path? That'd keep these settings contained in the test and potentially make the image reusable for other filesystem checks.

Reply:
No, but I think I'd rather address this holistically in follow up than starting to build general purpose images for individual tests.  @dgerd has been doing this for the conformance testing, so it is possible that this can reuse that image.